intelmq/bin/intelmqctl.py

```diff
@@ -121,15 +121,16 @@ def __init__(self, runtime_configuration, logger, controller):
     def bot_run(self, bot_id, run_subcommand=None, console_type=None, message_action_kind=None, dryrun=None, msg=None):
         pid = self.__read_pidfile(bot_id)
         if pid and self.__status_process(pid):
-            paused = True
-            self.bot_stop(bot_id)
-        else:
-            paused = False
+            self.logger.warning("Main instance of the bot is running in the background. You may want to launch: intelmqctl stop {}".format(bot_id))
```

Comment:
I changed ```IntelMQProcessManager.bot_run``` method so that it's not problem anymore to run multiple instance of bot.
	If we run a bot that is already running, only warning ```"Main instance of the bot is running in the background. You may want to launch: intelmqctl stop {}"``` is printed. 
	However, if there is a reason for the background instance is rather paused and relaunched after debugging is finished, I commented all the code so that I can restore it easily.